src/magento2/app/code/Ves/Megamenu/view/frontend/layout/default.xml

```diff
@@ -27,17 +27,20 @@
 		<css src="Ves_Megamenu::css/font-awesome.min.css" />
 		<css src="Ves_Megamenu::css/animate.min.css" />
 		<css src="Ves_Megamenu::css/styles.css" />
+
+		<css src="Ves_Megamenu::css/custom.css" media="screen and (min-width: 768px)" />
+		<css src="Ves_Megamenu::css/custom-mob.css" media="screen and (max-width: 767px)"/>

```

Comment:
Is this the proper place to add custom CSS? Line 14 says that this file should not be modified.